MekHQ/src/mekhq/Utilities.java

```diff
@@ -369,17 +369,30 @@ public static File lastFileModified(String dir, FilenameFilter filter) {
         CampaignOptions options = campaign.getCampaignOptions();
         ArrayList<String> variants = new ArrayList<>();
         for(MechSummary summary : MechSummaryCache.getInstance().getAllMechs()) {
-            // If this isn't the same chassis, is our current unit, or is a different weight we continue
-            if(!en.getChassis().equalsIgnoreCase(summary.getChassis())
+            // If this isn't the same chassis, is our current unit, we continue
+            if (!en.getChassis().equalsIgnoreCase(summary.getChassis())
                     || en.getModel().equalsIgnoreCase(summary.getModel())
-                    || summary.getTons() != en.getWeight()
                     || !summary.getUnitType().equals(UnitType.determineUnitType(en))) {
```

Comment:
Can we swap this to use UnitType.getTypeName(e.getUnitType()), as the current method is deprecated.

Reply:
Done.